api/objects.proto

```diff
@@ -257,7 +257,7 @@ message NetworkAttachment {
 	repeated string aliases = 3;
 	
 	// Map of all the driver options for this network
```

Comment:
Please update this comment.

Keeping the field comments correct and up to date is important. It helps us to document these fields and make sure they are operating correctly.